lib/logging/addon/components/logging/target-kafka/component.js

```diff
@@ -42,39 +51,58 @@ export default Component.extend({
       }
     },
   },
-  enableSSLConfig: computed('brokerType', 'config.brokerEndpoints.[].endpoint', function() {
-    const brokerType = get(this, 'brokerType')
 
-    if (brokerType === 'zookeeper') {
-      this.clearCA()
+  // cache and restore
+  brokerTypeChange: observer('brokerType', function() {
+    const t = get(this, 'brokerType');
+    const brokerEndpoints = get(this, 'brokerEndpoints');
+    const zookeeperEndpoint = get(this, 'config.zookeeperEndpoint');
+    const cachedZookeeperEndpoint = get(this, 'cachedZookeeperEndpoint');
+    const cachedBrokerEndpoints = get(this, 'cachedBrokerEndpoints');
```

Comment:
brokerEndpoints, cachedZookeeperEndpoint, cachedBrokerEndpoints can all be destructured.